index.css

```diff
@@ -284,4 +287,67 @@ body{
          display: none !important;
      }
  }
+  .hamburger {
+     color: #888;
+     cursor: pointer;
+     margin-top: 8px;
```

Comment:
Write `margin: {top} {right} {bottom} {left};` instead. Like `margin: 1px 2px 3px 4px;`. Same goes for padding and most other CSS properties.